### DISCUSSION TITLE
Packet rate modes naming standardization

### DISCUSSION THREAD

**Discussion by ajjjjjjjj** [ExpressLRS/ExpressLRS#1917] [Ideas]
Currently packet rates are inconsistently named, also on B/W 128x64 radios sensitivity is not visible for "full" modes.

Standard Lora modes are (for 2G4):
`50Hz, 100Hz Full, 150Hz, 250Hz, 333Hz Full, 500Hz`
FLRC rates use first letter to describe mode details, but no unit (Hz).
`D250, D500, F500, F1000`.

Idea is to use common pattern for both so Lora modes could become:
`L50, L100F, L150, L250, L333F, L500`

`[A][B][C]` where:
* [A] is L: Lora, F: FLRC, D: FLRC with repeated packet, like it already is,
* [B] is actual packet rate,
* [C] optional: F: Full channel mode.

This way we gain:
* consistent naming pattern,
* visible sensitivity on B/W radios,

```
#if defined(RADIO_SX127X)
    "L25(-123dBm);L50(-120dBm);L100(-117dBm);L100F(-112dBm);L200(-112dBm)";
#elif defined(RADIO_SX128X)
    "L50(-115dBm);L100F(-112dBm);L150(-112dBm);L250(-108dBm);L333F(-105dBm);L500(-105dBm);"
    "D250(-104dBm);D500(-104dBm);F500(-104dBm);F1000(-104dBm)";
```

What do you think?

**Comment by CapnBry** [comment]
I really do not like this system, it seems more cryptic than ever instead of making more sense. L100F? Also renaming them invalidates all existing information with very little benefit.

Users don't NEED to know that 150Hz uses LoRa. They don't need to care about it. Just leave it the way it is.

**Comment by ajjjjjjjj** [comment]
For me having `100Hz Full` and `D250` on the same list is more confusing. I would rather see `250Hz DVDA` if going this way.

**Comment by CapnBry** [comment]
Longer names don't fit on the screen of B&W radios as you said, we should just use the RFMD number! 🤡

**Comment by ajjjjjjjj** [comment]
I said that sensitivity does not fit (`100Hz Full(-`), DVDA would be visible. Having some options with it, and others without does not help much and going other way makes it at least less confusing.

I know that since looking at those options for many months everything may look very obvious, like who don't know what `D250` is? But for new users that list does not look good.

**Comment by CapnBry** [comment]
Well we also moved the "Hz" part of the mode name back to the left (before the sensitivity, it used to be `500(-105dBm)Hz`) because many users complained about not seeing the Hz part and that was too confusing. So dropping the Hz part seems to have already been decided to not be acceptable. We can't change all the mode names with every release as each person comes along with _their_ idea of how the information should be presented.
